Charts/fastcs/templates/pvcs.yaml

```diff
@@ -1,7 +1,7 @@
 # PVC for debugging volume
 {{ if .Values.editable }}
-{{- $location := default .Values.global.location .Values.location | required "ERROR - You must supply location or global.location" -}}
-{{- $ioc_group := default .Values.global.ioc_group .Values.ioc_group | required "ERROR - You must supply ioc_group or global.ioc_group" -}}
+{{- $location := default .Values.global.location .Values.location | default "" -}}
+{{- $ioc_group := default .Values.global.ioc_group .Values.ioc_group | default "" -}}
```

Comment:
Maybe we could set `is_ioc` to true here if `ioc_group` is not empty, so that it doesn't have to be a value? And so that we can't have `ioc_group=something` with `is_ioc=false`, which would probably do bad things.

Reply:
This sounds sensible to me.